GenderPayGap.WebUI/Views/ScopeNew/ConfirmOutOfScope.cshtml

```diff
@@ -0,0 +1,124 @@
+﻿@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using GovUkDesignSystem
+@using GenderPayGap.WebUI.Models.ScopeNew
+@using GenderPayGap.WebUI.Classes.Formatters
+@model GenderPayGap.WebUI.Models.ScopeNew.OutOfScopeViewModel
+
+@{
+    ViewBag.Title = "Confirm that your organisation is not in scope - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@{
+    string encryptedOrganisationId = Encryption.EncryptQuerystring(Model.Organisation.OrganisationId.ToString());
+    string reportingYearsString = Model.ReportingYear.Year + "/" + (Model.ReportingYear.Year + 1).ToString().Substring(2, 2);
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Your Organisations",
+                Href = Url.Action("ManageOrganisations", "Organisation")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.Organisation.OrganisationName,
+                Href = Url.Action("ManageOrganisation", "Organisation", new {id = encryptedOrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Change Scope (Reporting Year " + reportingYearsString + ")"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+@{
+    string reportingYearFormatted = Model.ReportingYear.Year + "/" + (Model.ReportingYear.Year + 1).ToString().Substring(2, 2);
+    string whyOutOfScopeFormatted = Model.WhyOutOfScope == WhyOutOfScope.Under250 ? "My organisation had fewer than 250 employees on the " + new GDSDateFormatter(Model.ReportingYear).FullStartDate
+        : "Other: " + Model.WhyOutOfScopeDetails;
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="POST" action="@Url.Action("ConfirmOutOfScopeAnswers", "ScopeNew", new {encryptedOrganisationId, reportingYear = Model.ReportingYear.Year })">
+            @(Html.AntiForgeryToken())
+            <h1 class="govuk-heading-xl">Review your information</h1>
+            
+            @(Html.HiddenFor(model => model.WhyOutOfScope))
+            @(Html.HiddenFor(model => model.WhyOutOfScopeDetails))
+            @(Html.HiddenFor(model => model.HaveReadGuidance))
+            @(Html.HiddenFor(model => model.ReportingYear))

```

Comment:
You don't need the hidden input for `ReportingYear` - it's a `[BindNever]` property so won't get bound anyway.
You've specified the actual reporting in the URL a few lines above here